controller/controller.js

```diff
@@ -12,5 +12,78 @@ exports.getApiTopics = (request, response) => {
     });
 };
 
+// /api/articles Endpoints
+exports.getApiArticles = (request, response) => {
+    // TODO: instead of doing this for every articles request, find way to just store comment count in the db and keep it updated, that way theres no need to calculate it each time
```

Comment:
Your controller shouldn't be doing this much manipulation, it's one priority is handling the req and res, all other leg work should be done outside of this